### PR TITLE
NEW workflow: WordPress.org README.md validation

### PR DESCRIPTION
**Pull request by carstingaxion**
This PR introduces a [new README validation workflow](https://github.com/marketplace/actions/wordpress-plugin-readme-md-validator) and a subtle fix to the existing readme.

> A GitHub Action that validates that the heading section of your plugin's README.md has newlines handled correctly.
>
> When writing markdown, two spaces are required at the end of a line to preserve single line breaks in the rendered view. Some people's (@jazzsequence) IDEs helpfully remove these and the diff can be missed in GitHub's PR view.

### Description of the Change

I had the feeling, that the readme.md is rendered weird on GitHub, so I looked for a validation and found this action.

The screenshots proofed my unclear feeling.

| BEFORE | AFTER |
|--------|--------|
| IMG_0209 | IMG_0210 | 

~~Closes #~~

### Changelog Entry

> Fixed - White space issues in readme.md (including a new automated test)

### Credits

Props @carstingaxion 

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/GatherPress/gatherpress/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my change.
- [x] All new and existing tests pass.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # GatherPress
 
-Stable tag: 0.30.0
-Tested up to: 6.6.1
-License: GPL v2 or later
-Tags: events, event, meetup, community
+Stable tag: 0.30.0  
+Tested up to: 6.6.1  
+License: GPL v2 or later  
+Tags: events, event, meetup, community
```
Comment (mauteri) [inline readme.md:6]: Not sure why this was changed, seems it's adding extra white space after text on lines 3 thru 6.

**Review by mauteri — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (carstingaxion) [reply readme.md:6]: Can the PR description maybe answer your question?

Comment (carstingaxion) [reply readme.md:6]: Because this is intentional and exactly by purpose of this PR.

Comment (mauteri) [reply readme.md:6]: Oh I see, yes, sorry. 

Issue I think might be that a lot of IDEs are set to remove extra trailing white space. Let me check out the branch later and test that. If it's an issue we can test another way of doing same thing.

Comment (mauteri) [reply readme.md:6]: Actually, this is fine. Just tested. Nice job!

**Review by mauteri — APPROVED** [review]

**Pull request merged**